.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
```

Comment:
Something is wrong, this diff shouldn't be showing up since this file already appears in my master.